### ISSUE TITLE
setext_with_atx_closed not available in Virtual Studio Code extension

### ISSUE THREAD

**Issue by RickTheGeezer** [DavidAnson/markdownlint#14]
I just installed the markdownlint extension in VS Code. I was going nuts trying to figure out why I couldn't get the .json config to take setext_with_atx_closed, when all the other options (including setext_with_atx) worked fine. So I stumbled on your commit history and see you've only just implemented it. I assume, then, that this change has not been migrated into the VS Code extension. Will you be updating the extension soon, or should it be already working?

To reproduce the symptom:

- Create file .markdownlint.json in project folder with the following content:
```
{
    "MD003": { "style": "setext_with_atx_closed" }
}
```
- In VS Code with markdownlint extension installed, open the project folder.
- Create a new file, set its language to Markdown, and enter the following lines:
```
Test of VS Code 'markdownlint' Extension
==

Setext Header
--

## ATX Header

### ATX_Closed Header ###

Some text.
```
- Observe that all headers are flagged with warnings for rule MD003. Presumably, the `setext_with_atx_closed` value for the `style` property is unknown to the extension.

**Comment by DavidAnson** [comment]
Your theory is correct: support for setext_with_atx_closed is not yet part of a markdownlint release (and therefore not part of a vscode-markdownlint release).

More context: The most recent tag and "Update to version x.y.z" commit correspond to 0.1.1 which is the latest package in npm. setext_with_atx_closed was added after that and therefore not available yet.

I'll update soon and this will show up in vscode-markdownlint. Sorry for the trouble!

**Comment by RickTheGeezer** [comment]
Very good, thank you!

Just so you know, the reason I missed that this option wasn't released yet was that I followed the link from the VS Code warning light bulb straight to __Rules.md__ in this repo, which of course you've already updated. This being my first time ever on GitHub (or BitBucket, SourceForge, etc.–I'm a dinosaur), it didn't occur to me that this could be a project in active development and the help link might go to not-yet-released documentation. I suppose such leading-edge-of-development glitches are commonplace for developers today, though, so I'll know better next time.

**Comment by DavidAnson** [comment]
Thanks for the details. What I'll do with the next vscode-markdownlint release is lock it to the version of Rules.md that corresponds to the version of markdownlint it's using. Sorry for not thinking to do so in the first place!
